packages/studio-be/src/core/app/inversify/app.inversify.ts

```diff
@@ -74,6 +74,11 @@ container
   .to(ConfigProvider)
   .inSingletonScope()
 
+container
+  .bind<MonitoringService>(TYPES.MonitoringService)
+  .to(CEMonitoringService)
+  .inSingletonScope()
```

Comment:
The monitoring service has absolutely nothing to do with the studio, it should stay in the core

Reply:
yep, leftover, I tried bringing back the `RealtimeService` in studio

I forgot to remove